src/NServiceBus.Core/Routing/MessageDrivenSubscriptions/ISubscriptionStorage.cs

```diff
@@ -17,5 +17,11 @@ public interface ISubscriptionStorage
         /// Unsubscribes the given client address from messages of the given types.
         /// </summary>
         Task Unsubscribe(string client, IEnumerable<MessageType> messageTypes, SubscriptionStorageOptions options);
+
+        /// <summary>
+        /// Returns a list of addresses of subscribers that previously requested to be notified
+        /// of messages of the given message types.
```

Comment:
I know you just copied it but the text doesn't read well?

How about "Returns a list of addresses for subscribers currently subscribed to the given `message type`" ?

@SzymonPobiega since you're neck deep in routing do you have something better?